lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/oauth/OAuthClientAuthorization.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
can you think of a name that differentiates from the existing "OAuthAuthorizationResponse" class, e.g. "OAuthClientAuthorizationRecord", "OAuthClientAuthorizationHistory"/